modules/test/playwright/tests/product-navigation-user-personal-bar-web/userPersonalBar.spec.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {userPersonalBarPagesTest} from '../../fixtures/userPersonalBarPagesTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	userPersonalBarPagesTest,
+	loginTest
+);
+
+test('LPD-15423 notification badge is not visible when show notification badge in personal menu is disabled', async ({
+	apiHelpers,
+	userPersonalBarPage,
+}) => {
+	await userPersonalBarPage.goToProcessBuilderConfigurationTab();
+	await userPersonalBarPage.enableSingleApproverWorkflowProduct();
+	await userPersonalBarPage.disableNotificationBadgeInPersonalMenu();
+
+	const catalog = await apiHelpers.headlessCommerceAdminCatalog.postCatalog(
+		'Test Catalog'
+	);
+
+	const product = await apiHelpers.headlessCommerceAdminCatalog.postProduct(
+		catalog.id,
+		'Test Product'
+	);
+
+	const response = await apiHelpers.headlessCommerceAdminCatalog.patchProduct(
+		product.productId
+	);
+
+	expect(response.ok()).toBe(true);
+
+	await userPersonalBarPage.goto();
+
+	await expect(userPersonalBarPage.notificationBadge).not.toBeVisible();
+
+	await userPersonalBarPage.disableSingleApproverWorkflowProduct();
+	await userPersonalBarPage.enableNotificationBadgeInPersonalMenu();
+
+	await apiHelpers.headlessCommerceAdminCatalog.deleteProduct(
+		product.productId
+	);
+
+	await apiHelpers.headlessCommerceAdminCatalog.deleteCatalog(catalog.id);
+});
+
+test('LPD-15423 notification badge is visible when show notification badge in personal menu is enabled', async ({
```

Comment:
do we need two separated tests? I mean... can we collapse them in one like
1) enablesingleworkflowproduct
2) create product
3) verify the notification badge is visible
4) disable the notification badge
5) verify the notification badge is not visible anymore
6) delete